### ISSUE TITLE
reserve space 

### ISSUE THREAD

**Issue by ChrisTsou** [TimUntersberger/nog#209]
I didnt see another issue bringing it up so here it goes: 
an option to make the bar reserve desktop space would be nice. like the windows taskbar does.

**Comment by ramirezmike** [comment]
By "reserve desktop space" are you referring to the position of the icons on the desktop?

**Comment by ChrisTsou** [comment]
yes, mainly. but also unmanaged maximized windows shouldn't overlap with the bar.
